src/main/java/cat/udl/eps/softarch/learnphysics/config/ApplicationContext.java

```diff
@@ -0,0 +1,79 @@
+package cat.udl.eps.softarch.learnphysics.config;
+
+import cat.udl.eps.softarch.learnphysics.domain.Question;
+import cat.udl.eps.softarch.learnphysics.domain.Theory;
+import cat.udl.eps.softarch.learnphysics.domain.Topic;
+import cat.udl.eps.softarch.learnphysics.domain.Level;
+import cat.udl.eps.softarch.learnphysics.repository.LevelRepository;
+import cat.udl.eps.softarch.learnphysics.repository.TopicRepository;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.repository.query.QueryLookupStrategy;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Supplier;
+import java.util.logging.Logger;
+
+@Component
+public class ApplicationContext implements InitializingBean {
+
+    private static final Logger LOG = Logger.getLogger(String.valueOf(ApplicationContext.class));
+
+    @Value("${levels}")
```

Comment:
This would be to get the value from the file `application.yml`, I don't think it's necessary here.